.github/workflows/golangci-lint.yml

```diff
@@ -20,3 +20,4 @@ jobs:
         with:
           version: latest
           args: --timeout=5m
+            # skip-cache: true
```

Comment:
Why add this just to comment it out? Why not just leave this out?

Reply:
I may want to re-enable it in the future